config/config.exs

```diff
@@ -2,4 +2,6 @@ import Config
 
 config :next_ls, :indexing_timeout, 100
 
+config :logger, :default_handler, config: [type: :standard_error]
```

Comment:
can you point me to the docs on this config option? I can't seem to find it in the OTP docs